CMakeLists.txt

```
@@ -2670,7 +2669,7 @@ endif()
 option(DEBUG_ASSERTIONS_FATAL "Fail if debug become true assertions" OFF)
 if(DEBUG_ASSERTIONS_FATAL)
   target_compile_definitions(mixxx-lib PUBLIC MIXXX_DEBUG_ASSERTIONS_FATAL MIXXX_DEBUG_ASSERTIONS_ENABLED)
-  if(QML)
+  if(QT6)
```

```suggestion
  if(QML)
```

Why did you change this? QT6 and QML are different options.

---

Only in 2.5 / main